tests/test_python3.py

```diff
@@ -74,13 +74,14 @@ def test_python3_with_public_docker():
     test_dir = "./temp_test_python3/with_public_docker"
     os.makedirs(f"{test_dir}", exist_ok=True)
     toolchest.python3(
-        tool_args="./input/example.fastq",
-        script="s3://toolchest-integration-tests/write_test.py",
-        inputs="s3://toolchest-integration-tests/example.fastq",
+        tool_args="",
```

Comment:
Legit question: is `tool_args=""` needed?

Reply:
No it's the default arg and can be removed